udata_schema_gouvfr/views.py

```diff
@@ -5,21 +5,52 @@
 from udata import theme
 from udata.frontend import template_hook
 
+import requests
+
 blueprint = Blueprint('schema', __name__, template_folder='templates')
 
 
-def validata_url(resource):
+def validata_url(resource, schema_url=None):
     base = current_app.config.get('SCHEMA_GOUVFR_VALIDATA_URL')
-
-    query = urlencode({
-        "input": "url",
-        "schema_name": f"schema-datagouvfr.{resource.schema}",
-        "url": resource.url,
-    })
+    if schema_url:
```

Comment:
You can mutualize {input, url} and add {schema_url} or {schema_name} based on your `if`.

Reply:
done